src/main/java/dk/kb/cdx/config/ServiceConfig.java

```diff
@@ -79,37 +43,22 @@ public static synchronized void initialize(String configSource) throws IOExcepti
         
         CDX_SERVER_URL=serviceConfig.getString("config.workflow.cdx_server_url");
         WARCS_INPUT_LIST_FILE=serviceConfig.getString("config.workflow.input_file");
-        WARCS_OUTPUT_LIST_FILE=serviceConfig.getString("config.workflow.output_file");
+        WARCS_OUTPUT_LIST_FILE=serviceConfig.getString("config.workflow.output_file");       
         THREADS=serviceConfig.getInteger("config.workflow.threads");
-        
+        DRYRUN=serviceConfig.getBoolean("config.workflow.dry_run");
```

Comment:
Do not like it. It can only happen if users delete it from the yaml file that comes with the distribution.
Can be very serious if they belived to have started it with 48 threads and default is 4.